### ISSUE TITLE
[Accessibility][DatePicker] Datepicker cannot be expanded by Alt+Down arrow

### ISSUE THREAD

**Issue by stacymiller96** [microsoft/fluentui#11508]
### Environment Information

- **Package version(s)**: 6.195.1
- **Browser and OS versions**: all

### Describe the issue:

Our product uses the DatePicker control that also allows for text input. Since the component is both a textfield and calendar callout, the component was identified by our accessibility testers as a combobox, which should be expandable from the textfield using Alt+Down arrow key. 

This was filed as an MAS bug on our product.

### Please provide a reproduction of the issue in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/7.74.2/#/examples/datepicker

#### Actual behavior:

DatePicker calendar callout can only be opened from the textfield using enter key or click. 

#### Expected behavior:

DatePicker calendar callout can be opened from the textfield using enter key, click, and Alt+Down arrow key. 

### Documentation describing expected behavior

Keyboard interaction documentation for textboxes within comboboxes: https://www.w3.org/TR/wai-aria-practices-1.1/#textbox-keyboard-interaction

**Comment by micahgodbolt** [comment]
@lorejoh12 has this come up before? It is currently role="combobox", which would imply that (optional) keyboard interaction.

**Comment by lorejoh12** [comment]
We just missed this, I can't think of any reason we would have intentionally avoided doing this.
